roles/openshift_logging/templates/es.j2

```diff
@@ -95,6 +95,13 @@ spec:
               readOnly: true
             - name: elasticsearch-storage
               mountPath: /elasticsearch/persistent
+          readinessProbe:
```

Comment:
this is captured here https://github.com/openshift/openshift-ansible/pull/3733 but was lost... i'll merge the original one if you want to remove it here